DBZ-6023 [doc] Documentation for `surrogate-key` field (#4351)
debezium/debezium: documentation/modules/ROOT/partials/modules/all-connectors/con-connector-ad-hoc-snapshots.adoc

subset of the contents of the {data-collection}(s).
endif::[]

ifeval::['{context}' != 'mongodb']
|`surrogate-key`
|_N/A_

> | An optional string, which represents a column name that will be used during the snapshot process as a primary key of the {data-collection}(s).

Reviewer: ```suggestion
| An optional string that specifies the column name that the connector uses as the primary key of a {data-collection} during the snapshot process.
```
